content/07.pathogenesis.md

```diff
@@ -46,22 +46,30 @@ One severe example includes reports of cytokine storm-like responses in patients
 
 #### Reproduction Number and Dynamics of Transmission
 
-Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
-The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@isbn:9780691116174].
-The basic reproduction number, R~0~, is the expected number of new (secondary) infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
-R~0~ is a unitless number which in mechanistic models is a combination of parameters principally related to the transmission rate (rate of infection transmitting interactions for an infectious individual) and the infectious period [@doi:10.1126/science.abb5659; @isbn:9780691116174].
-A pathogen can invade a susceptible population only if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996].
-The effective reproduction number, R~t~, describes how the reproduction number may change over time, and is used to quantify deviations in R from R~0~ that would occur, for example, as some fraction of the population became infected or as interventions were put into place.
+Disease spread dynamics can be estimated using R~0~, the basic reproduction number, and R~t~, the effective reproduction number. 
+Thus, accurate estimates are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
+R~0~ is one of most critical epidemiological parameters [@isbn:9780691116174]; R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x]. 
+A simple mechanistic model used to describe infectious disease dynamics is a susceptible-infected-recovered (SIR) state-transition model. 
+In this formulation individuals move through three states: susceptible, infected, and recovered; two parameters, $\gamma$ and $\beta$, are specified, the rate at which the infectious recover, and the infection transmission rate, respectively. 
```

Comment:
This sentence read awkwardly for me. A potential rewording like below may help:

``` suggestion
In this formulation, individuals move through three states: susceptible, infected, and recovered; two parameters, $\gamma$ and $\beta$, specify the rate at which the infectious recover, and the infection transmission rate, respectively. 
```

Reply:
I'm not sure what the problem is with leaving the sentence:
"The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@isbn:9780691116174]."
It's almost exactly what a very good modern infectious disease modelling textbook says at the very beginning. Everyone talks about R but just because they often miss mentioning the timescale of infection doesn't mean that we should ignore it. If you just have R, all you know about the dynamics is whether the epidemic grows or doesn't (based on if R is greater than 1 or less than 1). If you have both R (unitless) & the timescale of infection then you have the dynamics with a scaled notion of time -- i.e. you can describe the dynamics with units.

Reply:
See previous comment, the way this sentence is phrased implies different types of models (e.g. Markov) than the most commonly referred to model when using term SIR (e.g. differential equation or "compartmental". So we can generalize better, you can use this sentence instead:
"In this formulation people are categorized as either susceptible, infected..."